src/api-service/dev-deploy.ps1

```diff
@@ -14,9 +19,15 @@ try {
     Set-Location __app__
     (New-Guid).Guid | Out-File onefuzzlib/build.id -Encoding ascii
     (Get-Content -path "requirements.txt") -replace 'onefuzztypes==0.0.0', './onefuzztypes-0.0.0-py3-none-any.whl' | Out-File "requirements.txt"
+    if ($version -ne "$null") {
```

Comment:
you can set the default value of $version to be "0.0.0" and remove the  version check here and below.
also the check should be $null not "$null"